index.html

```diff
@@ -36,3 +36,136 @@ <h3 class="intro">I'm Bunbee<br>Glad to see<br> you!</h3>
         <li class="fa fa-facebook fa-2x"></li>
        </ul>
     </section>
+
+    <section class="card-section">
+        <div class="card-image">
+            <img src="img/SnapshootPortfolio(1).png" alt="firstproject">
+        </div>
+        <p class="card-header">Tonic</p>
+        <ul class="card-links">
+            <li>CANOPY</li>
+            <li> Black End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p class="card-text">A daily selection of privately <br>personalised reads; no accounts or sign-ups required</p>
+        <div class="lang-section">
+           <ul class="card-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+        </div>
+        <button class="btn1">See Project</button>
```

Comment:
The button for all the projects has a different border color in the design. Please see the original design and compare it with yours from the screenshot below.
![btns](https://user-images.githubusercontent.com/35504149/130323300-e642cff2-818f-477c-a134-ee0256f81c56.png)